packages/amplify-provider-awscloudformation/src/__tests__/utils/auto-updates.test.ts

```diff
@@ -74,9 +79,34 @@ describe('showBuildDirChangesMessage', () => {
     setInCloudBackendDir = false;
     setInCloudBackendDirWithoutRequireVerification = false;
     setInBackendDir = false;
+
+    fsMock.existsSync.mockReturnValue(true);
   });
 
   describe('project does not have auth', () => {
+    beforeEach(() => {
+      setInCloudBackendDir = false;
+      setInCloudBackendDirWithoutRequireVerification = false;
+      setInBackendDir = true;
+
+      fsMock.existsSync.mockReturnValue(false);
+    });
+
+    it('does not call warn', async () => {
+      await showBuildDirChangesMessage();
+      expect(printerMock.warn).not.toBeCalled();
+    });
+  });
+
+  describe('project does not have auth 2', () => {
```

Comment:
Is this named right `existsSync` returns `true` means that auth is in backend cloud right?

Reply:
this is the behavior that was previously tested, folder exists but auth not in the cloud, will update test names